CodeGen/templates/Msg.h

```diff
@@ -3,23 +3,37 @@
 
 #pragma once
 
+// UE
 #include <CoreMinimal.h>
```

Comment:
```suggestion
#include "CoreMinimal.h"
```
Just to make it consistent with all UE header inclusion